cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/BasicMapReduceContext.java

```diff
@@ -124,6 +129,14 @@ public String getWorkflowBatch() {
     return workflowBatch;
   }
 
+  /**
+   * @return the adapter name if run from within adapter, otherwise null
+   */
+  @Nullable
+  public String getAdapterName() {
```

Comment:
``null``, in the current design. Also, won't be added to the ``LoggingContext``, and in future, to the ``MetricsContext``, either